code/src/components/Album.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import Artist from './Artist'
+
+const Album = (props) => {
+  console.log(props)
```

Comment:
remember to always remove console.logs from your code after you finish debugging it.